test/Microsoft.Sbom.Api.Tests/Workflows/SbomParserBasedValidationWorkflowTests.cs

```diff
@@ -151,6 +151,8 @@ public async Task SbomParserBasedValidationWorkflowTests_ReturnsSuccessAndValida
         var rootFileFilterMock = new DownloadedRootPathFilter(configurationMock.Object, fileSystemMock.Object, mockLogger.Object);
         rootFileFilterMock.Init();
 
+        var osUtilsMock = new Mock<IOSUtils>(MockBehavior.Strict);
```

Comment:
Thank you for updating the tests when they exist! Hopefully eventually we'll be able to add much more test coverage to this repo.